src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty lines above the class declaration.